docs/guide/instrumentation.md

```diff
@@ -17,9 +17,12 @@ config.view_component.instrumentation_enabled = true
 Then subscribe to the `!render.view_component` event:
 
 ```ruby
-ActiveSupport::Notifications.subscribe("!render.view_component") do |*args|
-  event = ActiveSupport::Notifications::Event.new(*args)
+ActiveSupport::Notifications.subscribe("!render.view_component") do |event|
   event.name    # => "!render.view_component"
   event.payload # => { name: "MyComponent", identifier: "/Users/mona/project/app/components/my_component.rb" }
+
+  ApplicationComponent.logger.debug \
+    "  Rendered #{bold}#{event.payload[:name]}#{clear}" \
+    " (Duration: #{event.duration.round(1)}ms | Allocations: #{event.allocations})"
```

Comment:
I think we can remove these lines. Perhaps this could be added in another PR?
```suggestion
```

Reply:
Done! @joelhawksley you think we can merge? Or should be part of v3?